.ci/docker/common/install_cuda.sh

```diff
@@ -3,7 +3,7 @@
 set -ex
 
 NCCL_VERSION=v2.21.5-1
-CUDNN_VERSION=9.1.0.70
+CUDNN_VERSION=9.5.1.17
```

Comment:
No that it matters, but PR would have been 3 lines shorter, if global CUDNN_VERSION to be kept at 9.1, but 12.6 one updated to 9.5

@atalman , @seemethere : we need to pay attention to metadata and  make sure that poetry is still usable for cuda-12.6 if we keep different versions for different releases (I suspect right now it is not)

Reply:
@malfet Intentional, I want those lines deleted 1 by 1 in the future.

Reply:
@malfet Metadata for different system should be used from CUDA 12.4 - the one we publish to pypi. It should not affect poetry issue.